opm/autodiff/FlowMainEbos.hpp

```diff
@@ -561,6 +626,14 @@ namespace Opm
                                                   eclState(),
                                                   std::move( eclIO_ ),
                                                   Opm::phaseUsageFromDeck(deck())) );
+            // create output writer after grid is distributed, otherwise the parallel output
+            // won't work correctly since we need to create a mapping from the distributed to
+            // the global view
+            output_writer_.reset(new OutputWriter(grid(),
```

Comment:
This is the cause of failure. Seems to be duplicated, perhaps due to a merge error?

Moving from the same object twice is not a good idea... (null pointer ends up in the writer)